apps/condo/domains/resident/schema/RegisterResidentInvoiceService.js

```diff
@@ -141,7 +157,7 @@ const RegisterResidentInvoiceService = new GQLCustomSchema('RegisterResidentInvo
                 const ticket = await Ticket.create(context, {
```

Comment:
This mutation was created for paid tickets. So, we always create a ticket. Do we still need to create tickets always?